ui/app/AppLayouts/stores/RootStore.qml

```diff
@@ -52,7 +52,7 @@ QtObject {
 //    property var walletModelInst: walletModel
     property var userProfileInst: userProfile
 
-    property var accounts: walletSectionAccounts.model
+    property var accounts: walletSectionSendInst.accounts
```

Comment:
Here it won't work, I can't see you've introduced `walletSectionSendInst` in this component anywhere

Reply:
Look at line 70